src/test/util/net.h

```diff
@@ -108,6 +109,22 @@ class StaticContentsSock : public Sock
 
     int Connect(const sockaddr*, socklen_t) const override { return 0; }
 
+    std::unique_ptr<Sock> Accept(sockaddr* addr, socklen_t* addr_len) const override
+    {
+        if (addr != nullptr) {
+            // Pretend all connections come from 5.5.5.5:6789
+            memset(addr, 0x00, *addr_len);
+            if (*addr_len >= sizeof(sockaddr_in)) {
+                *addr_len = sizeof(sockaddr_in);
+                sockaddr_in* addr_in = reinterpret_cast<sockaddr_in*>(addr);
+                addr_in->sin_family = AF_INET;
+                memset(&addr_in->sin_addr, 0x05, sizeof(addr_in->sin_addr));
+                addr_in->sin_port = htons(6789);
```

Comment:
They are not used anywhere else. Also, a legit code should expect an incoming connection to arrive from any address, so think it would be bad to expose this, so that some code starts to depend on it or checks whether the accepted connection came from `5.5.5.5`.